### DISCUSSION TITLE
How to migrate website to new server in v4?

### DISCUSSION THREAD

**Discussion by xuanht** [EasyEngine/easyengine#1696] [General]
I find this link but it's not work https://easyengine.io/handbook/migrate-website/

Plz, tell me how do migrate website to new server in v4!

**Comment by freedog96150** [comment]
The easiest way is to use a plugin to backup/restore or migrate your site from one server to another.  If you are simply moving servers and keeping the URL the same then there is a specific order you should follow to assure that everything works as expected.  My favorite backup plugin is WPVIVID but this is by no means an endorsement of that plugin nor a suggestion to abandon one that you may already use and trust.  But that said, this is how I would go about it.

- Install plugin if not already installed.
- Make a complete site backup of your site.
- Download all backup files to your local computer.
- Setup your new server/VM and install Easyengine.
- Take your new IP address and assign it to the A record on your DNS server so that the domain name points to the new server.
- Check with https://whatsmydns.net or other DNS propagation checker to make sure that the records are updated with the new IP address.
- Use Easyengine to create your new site choosing the options you require.
- Once your site is live, log in and add the same backup/restore plugin you used on the original source site.
- Run the restore as per the instructions.

Do note that this is not the most efficient way to migrate a site, but I feel it is one of the easiest ways to help people achieve the migration with minimal issues.  Also note that this method will create some site downtime while the DNS updates and you install and restore your new server.  Migrating sites with zero downtime and in a faster manner is possible, but more complex.
